airflow/models/xcom.py

```diff
@@ -229,3 +229,16 @@ def deserialize_value(result) -> Any:
                       "for XCOM, then you need to enable pickle "
                       "support for XCOM in your airflow config.")
             raise
+
+
+def resolve_xcom_backend():
+    """Resolves custom XCom class"""
+    clazz = conf.getimport("core", "xcom_backend")
```

Comment:
should we default to `BaseXCom` here? Just in case, someone updates the code, but not the config.

Reply:
Thanks @msumit , good catch! I fixed it and extended tests.